cli/api.py

```diff
@@ -187,27 +187,26 @@ def generate_test_positivity(test_positivity_all_methods: pathlib.Path):
     help="Output directory for artifacts",
     type=pathlib.Path,
 )
-@click.option("--aggregation-level", "-l", type=AggregationLevel)
+@click.option("--level", "-l", type=AggregationLevel)
 @click.option("--state")
 @click.option("--fips")
-def generate_api_v2(model_output_dir, output, aggregation_level, state, fips):
+def generate_api_v2(model_output_dir, output, level, state, fips):
     """The entry function for invocation"""
 
     # Caching load of us timeseries dataset
-    combined_datasets.load_us_timeseries_dataset()
+    us_timeseries = combined_datasets.load_us_timeseries_dataset()
 
     active_states = [state.abbr for state in us.STATES]
     active_states = active_states + ["PR", "MP"]
+    active_states = []
 
     # Load all API Regions
-    regions = combined_datasets.get_subset_regions(
-        aggregation_level=aggregation_level,
-        exclude_county_999=True,
-        state=state,
-        fips=fips,
-        states=active_states,
+    regions = list(
+        us_timeseries.iter_regions(
```

Comment:
Yeah, it'd be nicer to add `MultiRegionTimeseriesDataset.get_subset` instead of implementing the loop in iter_regions but not sure it really makes much practical difference.